### PR TITLE
Director/kids testing

### PR DESCRIPTION
**Pull request by Garo-A**
- Added parent first name, last name, email, and phone number to kids view.
- Added Parent and Counselor tester accounts for easy login
- Updated readme with tester account information for easy access

### CODE REVIEW

**Comment by TheTempas** [comment]
Looks good to me!

File: app/controllers/director/kids_controller.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def new
   def create
     @kid = Kid.new(kid_params)
     @parent_id = Parent.find_by(user_id: User.find_by(email: params[:parent_email]).id).id
+    # decadent13@campers.com
```
Comment (DerangedMind) [inline app/controllers/director/kids_controller.rb]: Looks like a stray comment - wanna remove this?

File: db/seeds.rb [hunk 2]
```diff
@@ -68,6 +68,25 @@
...
 end
 
+puts "Creating Test Counselor..."
+
+test_counselor_user = User.create(
+  first_name: 'Test Counselor',
+  last_name: "TEST",
+  email: "counselor@campzone.com",
+  password: "password",
+  role: 'counselor'
```
Comment (DerangedMind) [inline db/seeds.rb]: Stick with double quotes or single quotes, please! :-)

**Comment by Garo-A** [comment]
Lol yeah. I was using that to keep track of the emails ;p

File: db/seeds.rb [hunk 3]
```diff
@@ -68,6 +68,25 @@
...
+)
+
+test_counselor = Counselor.create(
+  user_id: test_counselor_user.id,
+  alias: "GodMode",
```
Comment (DerangedMind) [inline db/seeds.rb]: Since they're not a full admin, maybe DemiGodMode? :p

File: db/seeds.rb [hunk 4]
```diff
@@ -102,3 +121,24 @@
...
+
+test_parent = Parent.create(
+  user_id: test_parent_user.id,
+  address: "123 Test Street",
+  phone_number: "5141234567"
+)
+
+KidsParent.create(
+  parent_id: test_parent.id,
+  kid_id: 3
```
Comment (DerangedMind) [inline db/seeds.rb]: Are you assigning the 3rd kid to this parent? Does that mean that they will have two parents? Not a problem, just asking! :-)

File: db/seeds.rb (hunk 2 above)
Comment (Garo-A) [inline db/seeds.rb]: Gotacha!

File: db/seeds.rb (hunk 3 above)
Comment (Garo-A) [inline db/seeds.rb]: LOL

File: db/seeds.rb (hunk 4 above)
Comment (Garo-A) [inline db/seeds.rb]: Yeah! That's exactly what I want actually to test to see if my page would render both! :D

**Pull request merged**